platform/src/abstractions/Platform.Search/Validators/PostSearchRequestValidator.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using FluentValidation;
+
+namespace Platform.Search;
+
+[ExcludeFromCodeCoverage]
+public class PostSearchRequestValidator : AbstractValidator<SearchRequest>
+{
+    public PostSearchRequestValidator()
+    {
+        RuleFor(p => p.SearchText).NotNull()
+            .MinimumLength(3)
+            .MaximumLength(100);
+
+        RuleFor(x => x.OrderBy)
+            .Must(orderBy => orderBy == null || orderBy.Value == "asc" || orderBy.Value == "desc")
```

Comment:
[nitpick] Do we want to validate as case insensitive, or else `.ToLower()` later if Azure Search requires these values in lowercase?

Reply:
See also `Platform.Domain.Ranking.Sort` as potential abstraction pattern